Makefile

```
@@ -1,21 +1,32 @@
-VERSION:=`cat ./VERSION`
-COMMIT:=`git describe --dirty=+WiP --always 2> /dev/null || echo "no-vcs"`
+VERSION:=$$(cat ./VERSION)
+COMMIT:=$$(git describe --dirty=+WiP --always 2> /dev/null || echo "no-vcs")
+OUT:=$$(pwd)/out
 
-.PHONY: test coverage fmt clean
+.PHONY:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)